### DISCUSSION TITLE
We have reached the 10,000 star mark.

### DISCUSSION THREAD

**Discussion by deeprobin** [dotnet/runtime#75469] [General]
![10k](https://user-images.githubusercontent.com/10373452/189732537-66fdc931-f88d-469e-a9db-438071effeb3.png)

**Comment by tfenise** [comment]
Still behind [coreclr](https://github.com/dotnet/coreclr) or [corefx](https://github.com/dotnet/corefx).

**Reply by Joe4evr** [reply]
Those repos are effectively dead, tho.

**Reply by deeprobin** [reply]
Let ask Github support to transfer the stars 😂

**Reply by stephentoub** [reply]
Does number of stars matter for anything more than as a vanity metric?

**Reply by deeprobin** [reply]
There is a most stars ranking from GitHub :)
The higher we get, the more likely people will find this repository.
